### PR TITLE
Leonard nori walletconnect test wallet

### PR DESCRIPTION
**Pull request by aqualen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,9 +9,10 @@
     "crypto"
   ],
   "author": "WalletConnect <walletconnect.org>",
-  "homepage": "https://walletconnect.org",
+  "homepage": "http://localhost:3000/testwallet",
   "license": "LGPL-3.0",
   "scripts": {
+    "serve-start": "serve -s build",
```
Comment (scottynomad) [inline package.json]: what's this doing?

File: .env.local [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_INFURA_PROJECT_ID=57047f35d665403dbad08deee19ceee1
```
Comment (scottynomad) [inline .env.local]: Might be a better idea to inject this into the environment when we run it rather than committing it.

File: package.json (hunk 1 above)
Comment (aqualen) [reply package.json]: Over in our nori-cli serve.js, I use the following to start the testwallet:


```
    if (localWalletConnectService) {
      const port = process.env.PORT || 3081;
      process.env.PORT = port;
      console.log(`Firing up the wallet-connect test service`);
      console.log(`port=${port}`);
      console.log(`port=${port}`);
      const localWalletService = ChildProcess.spawn('yarn', ['serve-start'], {
        cwd: './walletconnect-test-wallet/',
        stdio: 'inherit', // Will use process .stdout, .stdin, .stderr
        shell: true,
      });
      children.push(localWalletService);
      delete process.env.PORT;
    }

```

Comment (scottynomad) [reply package.json]: Right, I guess the question should have been what is this new dependency bringing to the table that the existing create react app scripts don't.

My thinking is if we avoid adding dependencies to this fork then we avoid merge messes with yarn.lock which will make it significantly easier to keep this fork up to date.

File: .env.local (hunk 2 above)
Comment (aqualen) [reply .env.local]: Yeah, let me try to make that better.  I was following the README: 

https://github.com/WalletConnect/walletconnect-test-wallet/blob/master/README.md

Comment (aqualen) [reply .env.local]: I delete that file.

File: package.json (hunk 1 above)
Comment (aqualen) [reply package.json]: What's the right way to manage it?

**Pull request merged**